BitmapGenerator/BitmapGenerator.py

```diff
@@ -92,9 +92,9 @@ def __init__(self):
     self.width = 1024
     self.height = 1024
     self.buildDirection = "SI" # TODO: allow other build directions
-    self.slabSpacing = 0.016 # in mm, TODO: confirm this is good
+    self.slabSpacing = 0.03 # in mm, # DONE
     self.slabSpacing = 0.5
-    self.slabThickness = 1 # in mm, TODO: confirm this is good
+    self.slabThickness = 1 # in mm, TODO: confirm this is good - Not sure what this refers to
```

Comment:
The slab thickness is needed because we need to have some data to for the volume rendering.  It means that we will have some overlap between adjacent slices.  Ultimately we should switch to using the slice infrastructure instead of the volume rendering infrastructure.